### PR TITLE
Newsletter Onboarding: Change subscribers placeholder copies

### PR DESCRIPTION
**Pull request by lezama**
Before:
Screenshot 2023-05-05 at 10 36 54

After:
Screenshot 2023-05-05 at 10 36 22

Added translations capabilities too

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/subscribers/index.tsx [hunk 1]
```diff
@@ -52,9 +52,9 @@ const Subscribers: Step = function ( { navigation } ) {
 							) }
 							showSubtitle={ true }
 							emailPlaceholders={ [
```
Comment (simison) [inline client/landing/stepper/declarative-flow/internals/steps-repository/subscribers/index.tsx]: Do we need the `emailPlaceholders={}` attribute at all here? We could just let the form within onboarding default to the placeholder used inside the component? This kind of flexibility seems overly complicated for little gain.

We can then update the placeholders in the component if still need; yours seem more fun and quirky than originals.

**Review by simison — APPROVED** [review]
Good to go if you wanna deploy as-is, but also left a note (https://github.com/Automattic/wp-calypso/pull/76636#discussion_r1186126623) about simplifying.

File: client/landing/stepper/declarative-flow/internals/steps-repository/subscribers/index.tsx (hunk 1 above)
Comment (lezama) [reply client/landing/stepper/declarative-flow/internals/steps-repository/subscribers/index.tsx]: 👍 Done

**Review by simison — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7974843

Thank you @lezama for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7974843).

**Pull request merged**